inlong-agent/conf/agent.properties

```diff
@@ -53,6 +53,7 @@ thread.pool.await.time=30
 agent.local.ip=127.0.0.1
 agent.local.uuid=
 agent.local.uuid.open=false
+agent.enable.oom.exit=false
```

Comment:
maybe default true?

Reply:
> maybe default true?

At present, there is no relevant detection tool. It is better to temporarily set it as false, otherwise users will not perceive it if the agent exits.